gen3_util/files/cli.py

```diff
@@ -113,9 +113,9 @@ def _manifest_ls(config: Config, project_id: str, object_id: str):
 
 @manifest_group.command(name="upload")
 @click.option('--project_id', default=None, required=False, show_default=True,
-              help="Gen3 program-project", envvar='PROJECT_ID')
+              help="Gen3 program-project authorization", envvar='PROJECT_ID')
 @click.option('--restricted_project_id', default=None, required=False, show_default=True,
-              help="Gen3 program-project, additional access restriction", envvar='RESTRICTED_PROJECT_ID')
+              help="Gen3 program-project, additional authorization", envvar='RESTRICTED_PROJECT_ID')
```

Comment:
Not sure how to use this feature. I think it is necessary to put a brief user story example in the docs for how to use this `RESTRICTED_PROJECT_ID` envvar with the `gen3_util manifest upload` command

Reply:
Added documentation here: https://github.com/ACED-IDP/aced-idp.github.io/blob/feature/update-workflow/docs/ACED-data-upload.md#restrict-access-to-some-files